<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatView</class>
 <widget class="QWidget" name="ChatView">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1334</width>
    <height>750</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1334</width>
     <height>750</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>261</width>
      <height>69</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>Chat</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="indent">
     <number>100</number>
    </property>
   </widget>
   <widget class="QWidget" name="ChatView_1" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>72</y>
      <width>901</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="leftLayout" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="newConvo">
          <property name="styleSheet">
           <string notr="true">color : black;</string>
          </property>
          <property name="text">
           <string>New Conversation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sampleNameInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="addLandlordButton">
          <property name="text">
           <string>Start Conversation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="convo">
          <property name="styleSheet">
           <string notr="true">color: black;</string>
          </property>
          <property name="text">
           <string>Conversations</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="conversationList"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="rightLayout" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="backButton">
          <property name="text">
           <string>Back to Homepage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="messageArea"/>
        </item>
        <item>
         <widget class="QLineEdit" name="messageInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="sendButton">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="backButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/back.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closedBtn">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>5</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/close.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="logoutBtn">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>10</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/logout.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
